lib/commons/aria/index.js

```diff
@@ -193,15 +193,17 @@ lookupTable.globalAttributes = [
 	'aria-relevant'
 ];
 
-lookupTable.role = {
+const role = {
```

Comment:
This is a very large/hard to review file. Might be nice to split this up somehow, or even pull this stuff from a dependency.

No need to change now tho.